File: ngSupportLocal/src/app/models/yelp-rating.ts
import { YelpReview } from './yelp-review';
export class YelpRating {
  rating: string;
  yelpReviews: YelpReview;

  constructor(rating?: string, yelpReviews?: YelpReview) {
    this.rating = rating;
    this.yelpReviews = yelpReviews;
  }
}

File: JPASupportLocal/src/main/java/com/skilldistillery/supportlocal/entities/YelpRating.java
package com.skilldistillery.supportlocal.entities;

import java.util.List;

public class YelpRating {
	
	private String rating; 
	private List<YelpReview> reviews;
	
	public String getRating() {
		return rating;
	}
	public void setRating(String rating) {
		this.rating = rating;
	}
	public List<YelpReview> getReviews() {
		return reviews;
	}
	public void setReviews(List<YelpReview> reviews) {
		this.reviews = reviews;
	}  
	
	
	
	
	

}
